performant_pagination/pagination.py

```diff
@@ -103,7 +103,14 @@ def _object_to_token(self, obj):
             value = obj._meta.get_field(pieces[-1]).value_to_string(obj)
 
         # return our value
-        return b64encode(value)
+        if isinstance(value, text_type):
+            value = value.encode("utf-8")
+
+        value = b64encode(value)
+
+        if not isinstance(value, text_type):
+            value = b64encode(value).decode("utf-8")
```

Comment:
Seems like this will double encode things, since value was encoded above and would go through again here.